contracts/Registry.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.5.0;
+
+import "./RegistryAbstract.sol";
+
+
+contract Registry is RegistryAbstract {
+    mapping(address => ContributorInfo) contributors;
+
+    constructor(address[] memory _admins) RegistryAbstract(_admins) public {}
+
+    function registerContributors(
+        address[] memory wallets,
+        uint256[] memory allowed
+    ) public {
+        return _registerContributors(wallets, allowed);
+    }
+
+    function _registerContributors(
+        address[] memory wallets,
+        uint256[] memory allowed
+    ) internal {
+        require(wallets.length == allowed.length);
+        for (uint256 i = 0; i < wallets.length; ++i) {
+            if (wallets[i] != address(0)) {
```

Comment:
Consider: would it be beneficial to revert instead of just skipping in case of zero address? To me it seems like an explicit error (invalid data in input)?

Reply:
Yes